### PR TITLE
add es6 and gulp

### PR DESCRIPTION
**Pull request by festinalente91**
- add gulp, nodemon, babel. now you can use ES6
- just type "gulp server" to run server.

### CODE REVIEW

File: src/app/app.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+const urlencodedParser = bodyParser.urlencoded({ extended: false });
+
+const VIEW_PATH = './../../view';
+app.use(express.static(VIEW_PATH));
+
+app.get('/index.htm', (req, res) => {
+  res.sendFile(path.resolve(__dirname + "/" + VIEW_PATH + "/" + "index.htm" ));
+});
+
+app.listen(3000, () =>{
```
Comment (heartjigi) [inline src/app/app.js:16]: change port

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015"]
+}
```
Comment (heartjigi) [inline .babelrc]: Y no enter?

File: .gitignore [hunk 3]
```diff
@@ -37,4 +37,7 @@ jspm_packages
 .node_repl_history
 
 # webstorm
-.idea
\ No newline at end of file
+.idea
+
+# babel
+dist
```
Comment (heartjigi) [inline .gitignore]: Y no enter

File: gulpfile.babel.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import gulp from 'gulp';
+import babel from 'gulp-babel';
+import sourcemaps from 'gulp-sourcemaps';
+import rimraf from 'rimraf';
+import nodemon from 'gulp-nodemon';
```
Comment (heartjigi) [inline gulpfile.babel.js:5]: sort

File: gulpfile.babel.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+gulp.task('clean', () => rimraf.sync(SOURCE.DIST));
+
+gulp.task('build', ['clean'], () => {
+  return gulp.src(SOURCE.ALL)
+    .pipe(sourcemaps.init())
+    .pipe(babel())
+    .pipe(sourcemaps.write("."))
+    .pipe(gulp.dest("dist"))
+});
```
Comment (heartjigi) [inline gulpfile.babel.js]: where is test

File: gulpfile.babel.js (hunk 4 above)
Comment (festinalente91) [reply gulpfile.babel.js:5]: sort what?

File: gulpfile.babel.js (hunk 5 above)
Comment (festinalente91) [reply gulpfile.babel.js]: didn't attached yet.

File: src/app/app.js (hunk 1 above)
Comment (festinalente91) [reply src/app/app.js:16]: right. but maybe after setting up config in the next PR. :)

File: .babelrc (hunk 2 above)
Comment (festinalente91) [reply .babelrc]: updated

File: .gitignore (hunk 3 above)
Comment (festinalente91) [reply .gitignore]: updated

**Comment by heartjigi** [comment]
LGTM Thanks! P

File: package.json [hunk 6]
```diff
@@ -5,7 +5,8 @@
   "main": "server.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "start": "node ./src/server.js",
```
Comment (ochanje210) [inline package.json]: fix plz

**Review by ochanje210 — APPROVED** [review]

**Review by heartjigi — APPROVED** [review]

**Pull request merged**